packages/page_object/test/page_object_example_usage_test.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:page_object/page_object.dart';
+
+void main() {
```

Comment:
future note... might be worth an example that shows composition (where one FinderWrapper holds a reference to another FinderWrapper)

Reply:
This is a good idea, I will attempt to incorporate it.

Reply:
done